### PR TITLE
Disable x-datadog-tags header propagation

### PR DESCRIPTION
**Pull request by bwoebi**
### Readiness checklist
- [x] (only for Members) Changelog has been added to the release document.
- ~[ ] Tests added for this feature/bug.~

### Reviewer checklist
- [ ] Appropriate labels assigned.
- [ ] Milestone is set.
- [ ] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

File: tests/ext/integrations/curl/distributed_tracing_curl_copy_handle.phpt [hunk 1]
```diff
@@ -3,6 +3,7 @@ Distributed tracing headers propagate after curl_copy_handle()
 --SKIPIF--
 <?php if (!extension_loaded('curl')) die('skip: curl extension required'); ?>
 <?php if (!getenv('HTTPBIN_HOSTNAME')) die('skip: HTTPBIN_HOSTNAME env var required'); ?>
+<?php die('skip: disabled functionality: x-datadog-tags'); ?>
```
Comment (labbati) [inline tests/ext/integrations/curl/distributed_tracing_curl_copy_handle.phpt:6]: I suggest to leave the test active (as it tests many things) and just removing the tags part

**Review by labbati — CHANGES_REQUESTED** [review]
Small change to a test

File: tests/ext/integrations/curl/distributed_tracing_curl_copy_handle.phpt (hunk 1 above)
Comment (bwoebi) [reply tests/ext/integrations/curl/distributed_tracing_curl_copy_handle.phpt]: done

**Review by labbati — APPROVED** [review]
Thanks Bob

**Pull request merged**